docs/en/benchmark.md

```diff
@@ -46,6 +46,8 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
     <td align="center" colspan="2">fp32</td>
     <td align="center" colspan="2">fp16</td>
     <td align="center" colspan="2">int8</td>
+    <td align="center" colspan="2">JetsonNano2GB-fp32</td>
```

Comment:
In line 15 we said "NVIDIA tesla T4 tensor core GPU for TensorRT". Could we create a new section for each device and move the device to the title of the section?